src/test/java/org/alliancegenome/curation_api/bulkupload/fms/PhenotypeAnnotationBulkUploadFmsITCase.java

```diff
@@ -215,8 +266,17 @@ public void agmPhenotypeAnnotationClearSecondaryAnnotationFields() throws Except
 
 	@Test
 	@Order(7)
-	public void allelePhenotypeAnnotationAddSecondaryAnnotationFieldsUpdate() throws Exception {
-		//TODO: add test for update of primary allele annotation
+	public void allelePhenotypeAnnotationClearSecondaryAnnotationFields() throws Exception {
+		checkSuccessfulBulkLoad(phenotypeAnnotationBulkPostEndpoint, phenotypeAnnotationTestFilePath + "AF_01_all_fields_primary_allele_annotation.json");
+		
+		RestAssured.given().
+			when().
+			get(phenotypeAnnotationGetEndpoint + allelePaId).
+			then().
+			statusCode(200).
+			body("entity.phenotypeAnnotationSubject.modEntityId", is(allele)).
+			body("entity", not(hasKey("inferredGene"))).
+			body("entity", not(hasKey("assertedAllele")));
```

Comment:
you don't need to check that the assertedAllele key is missing as it shouldn't be possible to have this anyway